src/js/game.js

```diff
@@ -0,0 +1,28 @@
+class Game {
+  constructor(user, score) {
+    this.user = user;
+    this.score = score;
+  }
+
+  getUser() {
+    return this.user;
+  }
+
+  getScore() {
+    return this.score;
+  }
+
+  setUser(user) {
+    this.user = user;
+  }
+
+  setScore(score) {
+    this.score = score;
+  }
+}
+
+function defyJSLinter() {
```

Comment:
- [ ] Please make use of the arrow function instead of the function keyword. For more information on how the ES6 arrow function syntax works, see this [Mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).

Reply:
Thanks for your feedback @Ghiftee, It is now solved on commit f0d37df.